pkg/pool-weighted/contracts/WeightedPoolProtocolFees.sol

```diff
@@ -186,14 +176,11 @@ abstract contract WeightedPoolProtocolFees is BaseWeightedPool, ProtocolFeeCache
     }
 
     /**
-     * @dev Returns the amount of BPT to be minted to pay protocol fees on yield accrued by the Pool.
+     * @dev Returns the percentage of the Pool's supply which corresponds to protocol fees on yield accrued by the Pool.
      * Note that this isn't a view function. This function automatically updates `_athRateProduct`  to ensure that
      * proper accounting is performed to prevent charging duplicate protocol fees.
      */
-    function _getYieldProtocolFee(uint256[] memory normalizedWeights, uint256 preJoinExitSupply)
-        internal
-        returns (uint256)
-    {
+    function _getYieldProtocolFeesPoolPercentage(uint256[] memory normalizedWeights) internal returns (uint256) {
```

Comment:
Since we have more/multiple fee-related functions now, maybe group them together in a sensible order with a // Protocol Fees header comment?